src/m365/spfx/commands/project/project-upgrade/rules/FileAddRemoveRule.spec.ts

```diff
@@ -8,7 +8,10 @@ import { FileAddRemoveRule } from './FileAddRemoveRule.js';
 
 class FileAddRule extends FileAddRemoveRule {
   constructor(add: boolean) {
```

Comment:
`add` should also be wrapped in an object, rather than being a separate variable. Adjust in all other similar classes that inherit for a base rule class.